### ISSUE TITLE
Some of public repository is not visible

### ISSUE THREAD

**Issue by spartan4cs** [batnoter/batnoter#86]
**Describe the bug**
I am not able to see the some of public repo for which  i want to create notes

**Comment by vivekweb2013** [comment]
@spartan4cs is the repository present in your user account or within the organisation?

**Comment by fduschka** [comment]
I am having the same problem! 
I think what causes the issue is that the api is only returning the first 30 Repositories.

**Comment by vivekweb2013** [comment]
@fduschka yes you are right. I'll need to update the api endpoint to return all the available repositories.

**Comment by raguay** [comment]
I'm having the same issue. At least return the repositories that have been updated more recently. I have way over 30 repositories and I can't get it to take my note one. You could also see the notes repository is available and ask to use it.
